examples/add.rs

```diff
@@ -55,18 +55,39 @@ pub fn main() {
         Ok(result)
     });
 
-    // Get the first available device.
-    let device = *Device::all().first().unwrap();
+    // First we run it on CUDA if available
+    if let Some(nv_device) = 
+        Device::by_vendor(rust_gpu_tools::Vendor::Nvidia) {
```

Comment:
You could import `rust_gpu_tools::Vendor` to make that line a bit shorter (same for the ones below).

Reply:
Done, thanks